packages/drawer/src/views/Drawer.tsx

```diff
@@ -94,11 +94,23 @@ type Props = {
   gestureHandlerProps?: React.ComponentProps<typeof PanGestureHandler>;
 };
 
+/**
+ * Disables the pan gesture by default on Apple devices in the browser.
+ */
+function shouldEnableGesture(): boolean {
+  if (Platform.OS === 'web' && navigator) {
+    const isWebAppleDevice =
+      /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;
```

Comment:
What does `window.MSStream` for? Looks like Windows Phone specific?

Reply:
https://stackoverflow.com/a/9039885